src/components/views/UserPage.js

```diff
@@ -160,11 +164,10 @@ const UserPage = () =>{
 
     // MAP
     let map = null;
-    let position = [0, 0];  // default position
-    let zoomScale = 14;     // set zoom
     if(user){
-        position = [user.latitude, user.longitude]     // latitude, longitude
-        if(position[0]===0){
+        let zoomScale = calculateZoom(user.maxRange)
+        let position = [user.latitude, user.longitude] // latitude, longitude
```

Comment:
we can also save the long and lat in the localstorage when login/registration happens. than it only would be the location from the user itself.
Are we able to visit other userprofiles?

Reply:
Now, with specific access, it doesn't matter anymore. We don't visit other user profiles.